index.bs

```diff
@@ -3210,6 +3211,15 @@ A [=remote end=] has a <dfn>viewport overrides map</dfn> which is a weak map bet
 A [=remote end=] has a <dfn>locale overrides map</dfn> which is a weak map between
 [=navigables=] or [=user contexts=] and string.
 
+A <dfn>screen area</dfn> is a [=struct=] with an [=struct/item=] named
+<dfn attribute for="screen area">height</dfn> which is an integer,
+a [=struct/item=] named <dfn attribute for="screen area">width</dfn> which is an integer,
+a [=struct/item=] named <dfn attribute for="screen area">x</dfn> which is an integer,
+a [=struct/item=] named <dfn attribute for="screen area">y</dfn> which is an integer.
+
+A [=remote end=] has a <dfn>screen area overrides map</dfn> which is a weak map between
```

Comment:
nit: @jgraham used a struct to store the config for [session's extra headers](https://www.w3.org/TR/webdriver-bidi/#session-extra-headers). Consider using this pattern as well.